ffc-pay-alerts/notify/send-email.js

```diff
@@ -3,13 +3,17 @@ const {
   notifyApiKey,
   notifyEmailTemplateId,
   notifyEmailAddress
-} = require('./config')
+} = require('../config')
+
+const validateEmail = require('./validate-email')
 
 const NotifyClient = require('notifications-node-client').NotifyClient
 const notifyClient = new NotifyClient(notifyApiKey)
```

Comment:
Should this Notify config be outside this file?

Reply:
As discussed, dont think it needs to be split into seperate file.